en/docs/references/ballerina-expression-syntax.md

```diff
@@ -312,111 +312,110 @@ The following operators are available in the expression editor for numeric value
 
 #### Examples
 
-- Checking if the value of a variable (`firstNumber`) is greater than a literal.
+- Checks if the value of a variable (`firstNumber`) is greater than a literal:
 
   ```ballerina
   firstNumber > 10
   ```
   
-- Checking if the value of a variable (`firstNumber`) is less than or equal to the value of another variable (`secondNumber`).
+- Checks if the value of a variable (`firstNumber`) is less than or equal to the value of another variable (`secondNumber`):
 
   ```ballerina
   firstNumber <= secondNumber
   ```
 
 ## HTTP request related operations
 
-All the HTTP-related service applications created in Choreo have a variable called `request` (of type `http:Request`) which the developers can use to query information related to the HTTP request. Following is a list of such common use cases.
+All HTTP-related service applications created in Choreo have a variable called `request` (of type `http:Request`), which you can use to query information related to an HTTP request. Following is a list of such common use cases:
 
 ### Reading a header value in the request
-A typical HTTP request contains many headers.  These include standard and custom headers. You can use the `request` variable's `getHeader` function to read a specific header value in any expression-enabled input field.
+A typical HTTP request contains many headers.  These include standard and custom headers. You can use the request variable's `getHeader` function to read a specific header value in any expression-enabled input field.
 
 #### Examples
 
-- Reading the value of an HTTP header (`Access-Control-Allow-Origin`) passing the header name.
+- Reads the value of an HTTP header on passing the header name (`Access-Control-Allow-Origin`):
 
   ```ballerina
   check request.getHeader("Access-Control-Allow-Origin")
   
   ```
   
-- Reading the value of an HTTP header by passing a variable that contains the header name (`headerName`). Note the omission of the double-quotes.
+- Reads the value of an HTTP header on passing a variable that contains the header name (`headerName`). Note the omission of the double quotes:
 
   ```ballerina
   check request.getHeader(headerName)
   ```
 
 ### Checking whether a header is present in the request
 
-Applications sometimes need to check the availability of a header before proceeding to something like reading the header value. You can use the  `hasHeader`  function in the `request` object for this purpose.
+Applications sometimes need to check the availability of a header before proceeding to something like reading the header value. You can use the `hasHeader`  function in the request object for this purpose.
```

Comment:
Why are we removing backtick from `request`?

Reply:
Removed because in such occurrences `request` is not really the variable name. Does it make sense to use backticks in such occurences?

Reply:
Formatted with backticks.

Reply:
There is an extra space after `has header`?